### PR TITLE
Upgrde SonarQube to 7.8 and sonar-scala to 7.6.

### PR DESCRIPTION
**Pull request by mwz**
I also changed the following:
- removed SonarScala from the full image as it cannot co-exist with sonar-scala
- disabled SonarQube telemetry, which is enabled by default 👀
- formatted the readme files with prettier

### CODE REVIEW

File: vars.json [hunk 1]
```diff
@@ -15,6 +15,13 @@
   },
   "versions": {
     "current": [
```
Comment (sslavian812) [inline vars.json:17]: Why does this list say `current`, if it's in reality contains `old`/`previous` versions?

File: 3.7.0-full/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# SonarQube 7.8 image with bundled sonar-scala 7.6.0 (https://github.com/mwz/sonar-scala).
+
+FROM sonarqube:7.8-community
```
Comment (sslavian812) [inline 3.7.0-full/Dockerfile:3]: Are you sure you want to keep a separate folder for every old version you had, despite it's outdated? 
Would it be better just to change one Dockerfile upgrading some versions?

**Comment by sslavian812** [comment]
Btw, how about updating to `7.9 LTS`, since it's already out there?
https://www.sonarqube.org/whats-new/

They claim that they added Scala support in this release. Can we use it instead of `scapegoat`, `scoverage` and `scalastyle`?

File: 3.7.0-full/Dockerfile (hunk 2 above)
Comment (mwz) [reply 3.7.0-full/Dockerfile:3]: Yeah, I was planning to clean up this repo, so I'll do that at some point.

File: vars.json (hunk 1 above)
Comment (mwz) [reply vars.json:17]: all it means really is the non-lts versions, perhaps that's exactly what I should have named it

**Review by mwz — COMMENTED** [review]
> Btw, how about updating to `7.9 LTS`, since it's already out there?

I haven't upgraded sonar-scala to the latest LTS yet, but I'll get around doing that soon.

> They claim that they added Scala support in this release. Can we use it instead of `scapegoat`, `scoverage` and `scalastyle`?

They have their own Scala plugin called [SonarScala](https://www.sonarsource.com/products/codeanalyzers/sonarscala.html) while I maintain the community version [sonar-scala](https://github.com/mwz/sonar-scala) which comes with a slightly different feature set and works differently from the official plugin. Have a look at the comparison [here](https://github.com/mwz/sonar-scala/issues/179#issuecomment-477719803) to get a better idea about the differences.

**Pull request merged**